sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -41,18 +43,28 @@ public static DirectConnectionConfig getDefaultConfig() {
         return new DirectConnectionConfig();
     }
 
-    //  TODO: (DANOBLE) - To update these docs.
-
     /**
-     * Gets the direct connection timeout
+     * Gets the connection timeout for direct client,
+     * represents timeout for establishing connections with an endpoint.
+     *
+     * Configures timeout for underlying Netty Channel {@link ChannelOption#CONNECT_TIMEOUT_MILLIS}
+     *
+     * By default, the connection timeout is disabled, unless specified by user.
+     *
      * @return direct connection timeout
      */
     public Duration getConnectionTimeout() {
         return connectionTimeout;
     }
 
     /**
-     *  Sets the direct connection timeout
+     * Sets the connection timeout for direct client,
```

Comment:
It is the idleConnectionTimeout that defaults to zero. The connectTimeout defaults to the value of requestTimeout: 60 seconds prior to this PR.

Reply:
Cool, will add default value here - and will add the constant too..